spec/validation/checksum_spec.rb

```diff
@@ -0,0 +1,106 @@
+# frozen_string_literal: true
+require "spec_helper"
+
+describe HathiTrust::Validation::Checksums::Exists do
+  let(:mocked_sip) { HathiTrust::SIP::SIP.new("") }
+  let(:validation) { described_class.new(mocked_sip) }
+
+  describe "#validate" do
+    context "when checksum file exists in the package" do
+      before(:each) { allow(mocked_sip).to receive(:files).and_return([HathiTrust::SIP::CHECKSUM_FILE]) }
+
+      it "does not return any messages" do
+        returned_message = validation.validate
+        expect(returned_message).to be_nil
+      end
```

Comment:
This test is essentially, the shared example of "a validator with a valid package".  Here I'm testing a validation and not a validator so the shared example won't work because of a naming mismatch.